app/lib/actions/candlepin/candlepin_listening_service.rb

```diff
@@ -59,7 +59,7 @@ def start(suspended_action)
         else
           suspended_action.notify_not_connected("Not Connected")
         end
-      rescue TransportFailure => e
+      rescue => e
```

Comment:
@cpeters Probably not, @jlsherrill and I could not find where TransportFailure comes from (not katello, foreman, or qpid), curious to hear if anyone has some insight @thomasmckay do you know anything about this line?